### DISCUSSION TITLE
Capture the uninstaller listing as csv or xls output??

### DISCUSSION THREAD

**Discussion by ckknewsposter** [Klocman/Bulk-Crap-Uninstaller#549] [Ideas]
structured output that included eol/cr/lf character would be nice for install/reinstall/audit tracking of software changes.

Familiar with the copy to clipboard functionality.  Looking for an output option that is a bit more 'formatted' than pipe-delimited.

**Comment by Klocman** [comment]
You can export the whole list to xml from the File menu.

**Reply by ckknewsposter** [reply]
Thank You.  Completely missed that, my bad ;-(
